zeq/hbp/vocabulary.h

```diff
@@ -23,9 +31,46 @@ static const uint128_t EVENT_SELECTED_IDS(
     lunchbox::make_uint128( "hbp::SelectedIDsEvent" ));
 static const uint128_t EVENT_TOGGLE_ID_REQUEST(
     lunchbox::make_uint128( "hbp::ToggleIDRequest" ));
+static const lunchbox::uint128_t EVENT_REQUEST(
+    lunchbox::make_uint128( "hbp::RequestEvent" ));
+static const lunchbox::uint128_t EVENT_IMAGE_RAW(
+    lunchbox::make_uint128( "hbp::ImageRawEvent" ));
 //@}
 
 /**
+ * Serialize the given rgba raw image into an Event of type EVENT_IMAGE_RAW.
+ * The image format must be raw rgba with 8bits per component.
+ * @param image the image.
+ * @return the serialized event.
+ */
+::zeq::Event serializeImageRaw( const imageRaw& image );
+
+/**
+ * Deserialize the given EVENT_IMAGE_RAW event into an rgba raw image.
+ * The image format is raw rgba with 8bits per component.
+ * @param event the zeq EVENT_IMAGE_RAW.
+ * @return the image.
+ */
+const imageRaw deserializeImageRaw( const ::zeq::Event& event );
+
+/**
+ * Serialize the given request type into an Event of type EVENT_REQUEST.
```

Comment:
Here it says "request type" and in the prototype and the @param description it refers to an event type. Use the same nomenclature.